src/style.css

```diff
@@ -1,33 +1,43 @@
 /* stylelint-disable */
```

Comment:
- Please, kindly remove this `stylelint-disable` and try fixing the stylelint error as this helps you write readable and maintainable code, also this is a practice that companies now embrace in the industry. so to get comfortable, please, try fixing it. I'm making this `REQUIRED` because you've obviously disabled the linter checks for the styling. please fix this issue and ask for another review.
- Please, kindly also observe your footer as you've added [this forward slash](https://github.com/Solobarine/leaderboard/blob/a01efb071d17e0a4845681263ab7af81a9126ad2/src/index.html#L30) which is not a good user experience on your design. please kindly improve on this.